spring-integration-core/src/test/java/org/springframework/integration/aop/MethodAnnotationPublisherMetadataSourceTests.java

```diff
@@ -16,25 +16,26 @@
 
 package org.springframework.integration.aop;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.reflect.Method;
-import java.util.Map;
-
 import org.junit.Test;
-
 import org.springframework.core.annotation.AliasFor;
 import org.springframework.expression.Expression;
 import org.springframework.integration.annotation.Publisher;
 import org.springframework.messaging.handler.annotation.Header;
 import org.springframework.messaging.handler.annotation.Payload;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.reflect.Method;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.junit.Assert.*;
+import static org.springframework.test.util.ReflectionTestUtils.getField;
```

Comment:
Please, no asterisk imports. No, static imports for not allowed methods, like this `getField()`.

Also, please, try to keep the import order we support.

To be sure that you are right, please, run Gradle command: `/gradlew clean :spring-integration-core:check`.

This way you'll get tests passed and also Checkstyle rules checked.

Reply:
Updated the style.